### PR TITLE
bugfix: Fix exec and stop

### LINKED ISSUE #187: How to Run a container with Pouch ?
[root@VM_58_62_centos ~]# pouch images
IMAGE ID             IMAGE NAME                                               SIZE
be0cd392e45b         docker.io/library/hello-world:latest                     3074
f1ca87d9adb6         docker.io/library/nginx:alpine                           434
[root@VM_58_62_centos ~]# pouch create docker.io/library/nginx:alpine
container ID: 546d3d323c442dc637581441449d362c9e4d9afbf373e1e50cbc9385955e98c8, name: 546d3d 
[root@VM_58_62_centos ~]# pouch start 54
Error: failed to start container 54: {"message":"failed to create task, container id: 546d3d323c442dc637581441449d362c9e4d9afbf373e1e50cbc9385955e98c8: OCI runtime create failed: unable to retrieve OCI runtime error (open /var/lib/pouch/containerd/state/io.containerd.runtime.v1.linux/default/546d3d323c442dc637581441449d362c9e4d9afbf373e1e50cbc9385955e98c8/log.json: no such file or directory): exec: \"runc\": executable file not found in $PATH: unknown"}

failed to start container 54: {"message":"failed to create task, container id: 546d3d323c442dc637581441449d362c9e4d9afbf373e1e50cbc9385955e98c8: OCI runtime create failed: unable to retrieve OCI runtime error (open /var/lib/pouch/containerd/state/io.containerd.runtime.v1.linux/default/546d3d323c442dc637581441449d362c9e4d9afbf373e1e50cbc9385955e98c8/log.json: no such file or directory): exec: \"runc\": executable file not found in $PATH: unknown"}

------------------------------------------------------OR----------------------------------------------------------------------------
[root@VM_58_62_centos ~]# pouch images
IMAGE ID             IMAGE NAME                                               SIZE
be0cd392e45b         docker.io/library/hello-world:latest                     3074
f1ca87d9adb6         docker.io/library/nginx:alpine                           434
[root@VM_58_62_centos ~]# pouch start docker.io/library/nginx:alpine   
Error: failed to start container docker.io/library/nginx:alpine: {"message":"failed to get container info with prefix: docker.io/library/nginx:alpine, there are 0 containers"}

failed to start container docker.io/library/nginx:alpine: {"message":"failed to get container info with prefix: docker.io/library/nginx:alpine, there are 0 containers"}

------------------------------------------------------------------------------------------------------------------------------------------
I am a college Student. The Pouch-CLI Document is Wanted ~~ @allencloud @JacksonTian  @zhuzhaoyuan @jlusdy  #130

### PR DESCRIPTION
**Pull request by skoo87**
the pr contain two commits, one fix exec, another fix stop.

1. fix the timeout error of "pouch stop"

e.g.
    pouch stop d56e78b
    Error: failed to stop container d56e78: {"message":"failed to destroy container: d56e78b56a79846d77d6d5f4d16260715e30dd7ebb118dbef91f7c639485a3a9: time out"}

2. fix "pouch exec"

when pouch exec a process that not found in $PATH in container, the
    pouch cli will hang forever, don't print any error. the patch will fix
    it, and print error on STDOUT.

fix #187 

 Signed-off-by: skoo87 <marckywu@gmail.com>

### CODE REVIEW

File: daemon/mgr/container.go [hunk 1]
```diff
@@ -473,8 +473,12 @@ func (cm *ContainerManager) stoppedAndRelease(id string, m *ctrd.Message) error
 }
 
 func (cm *ContainerManager) exitedAndRelease(id string, m *ctrd.Message) error {
-	// release io.
 	if io := cm.IOs.Get(id); io != nil {
+		if err := m.StartError(); err != nil {
+			fmt.Fprintf(io.Stdout, "%v\n", err)
```
Comment (allencloud) [inline daemon/mgr/container.go]: This is will stream log into stdout. But I think it is OK to stream into log, logrus.

If this is Pouchd's log, it is better to use a unified way to collect log, currently `logrus`.

WDYT? @skoo87

Comment (skoo87) [reply daemon/mgr/container.go]: NO, it is not log,  it's that send the error message to CLI's stdout.

Comment (allencloud) [reply daemon/mgr/container.go]: Is `io.Stdout` the system stdout of Pouchd?
I am a little confused.

Comment (skoo87) [reply daemon/mgr/container.go]: No, it's not system stdout,  you can think it's a pipe be used to connect to 'CLI''s STDOUT.

Comment (allencloud) [reply daemon/mgr/container.go]: Oh, I get it. I think `io` makes me confused. I mistake `io` as `os`.
Could you change name io to be a more readable variable?

Comment (skoo87) [reply daemon/mgr/container.go]: the said have a reason.  But if you look at the context and understand it, I think you will not confused, believe me the "io" is very readable, ^_^.

**Comment by skoo87** [comment]
ping @allencloud

**Comment by allencloud** [comment]
LGTM

**Pull request merged**